### ISSUE TITLE
Possible policy misconfiguration on Unpkg.com CDN is breaking student projects 

### ISSUE THREAD

**Issue by golanlevin** [ozramos/handsfree#3]
Dear Oz, 
I hope you are well. I'm writing because there appears to be a possible policy misconfiguration on the Unpkg.com CDN causing an error 502 for the Handsfree.js 8.5.1 files. 

> Access to fetch at 'https://unpkg.com/handsfree@8.5.1/build/lib/assets/@mediapipe/hands/hands.binarypb' from origin 'https://preview.openprocessing.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
hands.js:59         
> GET https://unpkg.com/handsfree@8.5.1/build/lib/assets/@mediapipe/hands/hands.binarypb net::ERR_FAILED 502

<img width="654" alt="Screen Shot 2023-03-15 at 8 13 29 PM" src="https://user-images.githubusercontent.com/290053/225476583-9bd36eda-58c5-4456-9b61-fb7175f367c7.png">

Currently my students are using this (older, stable) version of Handsfree.js; the policy change has broken all of their projects. I would be so grateful if there was anything you could do to repair this hopefully small problem. You can test the issue with my template apps, here: 
https://openprocessing.org/user/6660/?view=sketches&o=82

![ezgif-1-3a3ef9fb67](https://user-images.githubusercontent.com/290053/225476924-bea127d2-7e2a-4d7b-864c-fb5f6d3a85ac.gif)

![ezgif-5-266fe8105f](https://user-images.githubusercontent.com/290053/225476932-12d32de5-4ad2-432b-bdfe-68a128521613.gif)

Thank you so much for your consideration Oz!
Yours as ever, 
Golan

**Comment by ozramos** [comment]
Hi Golan, it's so great to hear from you! I haven't changed anything on npm in quite some time, and I just checked all the templates and they work great!

Can you try again and let me know if you're still experiencing this? This repo actually contains the latest version of what's on npm (which is what unpkg pulls from) so I'm hoping it was just a brief server hiccup

**Comment by golanlevin** [comment]
You're right Oz! It was just a temporary server glitch. Since I saw you revisiting this project, I thought maybe you had been tinkering with the Unpkg, but it looks like the problem was a pure coincidence. 

Please feel free to hit me up if there's ever any way I can help you. And, **TAKE IT EASY** my friend. <3

<img width="625" alt="Screen Shot 2023-03-15 at 9 56 54 PM" src="https://user-images.githubusercontent.com/290053/225490214-024ea550-0977-46fd-b6df-a35252ef3042.png">
